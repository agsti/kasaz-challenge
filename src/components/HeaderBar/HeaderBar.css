.Header {
    height: 60px;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    z-index: 2;
    position: relative;
    background-color: white;
}

.header-item {
    margin-left: 10px;
    color: white;
}

.kasaz-logo {
    height: 40px;
}

