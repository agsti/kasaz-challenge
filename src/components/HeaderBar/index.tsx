import React from "react";
import HamburgerIcon from "./icons/HamburgerIcon.svg";
import KasazLogo from "./icons/KasazLogo.svg";

import "./HeaderBar.css";

export default function HeaderBar(){
    return <header className="Header">
        <span className="header-item">
            <img src={HamburgerIcon} />
        </span>
        <span className="header-item">
            <img src={KasazLogo} className="kasaz-logo" />
        </span>
        <div className="header-item"></div>
        </header>
}
