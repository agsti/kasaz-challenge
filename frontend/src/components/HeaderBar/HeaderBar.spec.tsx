import React from 'react';
import { shallow} from 'enzyme';
import {describe, expect, it} from '@jest/globals';

import HeaderBar from './';

describe('HeaderBar', () => {
  it('should render correctly', () => {
      const component = shallow(<HeaderBar />);
  
    expect(component).toMatchSnapshot();
  });
});
