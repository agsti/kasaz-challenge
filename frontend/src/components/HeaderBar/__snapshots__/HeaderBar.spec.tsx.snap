// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBar should render correctly 1`] = `
<header
  className="Header"
>
  <span
    className="header-item"
  >
    <svgrLoader
      fill="#41af68"
    />
  </span>
  <span
    className="header-item"
  >
    <svgrLoader
      className="kasaz-logo"
    />
  </span>
  <div
    className="header-item"
  />
</header>
`;
